src/editor/editor.css

```diff
@@ -2,25 +2,47 @@
   padding: 5px;
 }
 
+.ProseMirror {
+  isolation: isolate;
```

Comment:
Never heard of this before! What's this used for?

Reply:
Creates a new stacking context. It'll keep any negative z-indexed child elements from slipping behind it.